client/state/post-formats/schema.js

```diff
@@ -0,0 +1,19 @@
+export const items = {
+	type: 'object',
+	additionalProperties: false,
+	patternProperties: {
+		// Site ID
+		'^\\d+$': {
+			type: 'object',
+			description: 'List of supported post formats.',
+			additionalProperties: false,
+			patternProperties: {
+				// ID of the post format
+				'^[0-9a-z]+$': {
```

Comment:
This might be a little too strict. Can post formats have dashes or underscores in their name?